static/styles/user_status.css

```diff
@@ -29,9 +29,12 @@
             .selected_emoji {
                 width: 20px;
                 height: 20px;
-                top: 50%;
-                transform: translateY(-50%);
                 cursor: pointer;
+                /*
+                    the following rule is not necessarily better than the one
+                    from a6bef5154197b15c20445526fd3f219b4f2e5379 but it works.
+                */
+                top: -2px;
```

Comment:
Does this interact well with zooming the browser window?  I'd generally expect that to be an advantage of explicit vertical centering styling.  like what was there before.  (Though I feel like we might be able to use more modern CS alignment things to just say "please center this vertically")

Reply:
Also I don't quite understand under what circumstances this CSS is used -- can you explain?

Reply:
OK, I'm not sure why the -2px should be correct, but I guess this works.